### PR TITLE
[Site] Add attribute `xmlns` to icons

### PR DESCRIPTION
**Pull request by seb-jean**
I allow myself to add the `xmlns="http://www.w3.org/2000/svg"` attribute.

What do you think?

### CODE REVIEW

**Comment by smnandre** [comment]
~~As this would potentially change how icons are renders in existing apps... what about updating the recipe instead ?~~

Read too quickly sorry!

File: ux.symfony.com/config/packages/ux_icons.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ux_icons:
     default_icon_attributes:
         class: 'Icon'
+        xmlns: 'http://www.w3.org/2000/svg'
```
Comment (smnandre) [inline ux.symfony.com/config/packages/ux_icons.yaml]:
```suggestion
        xmlns: 'https://www.w3.org/2000/svg'
```

**Comment by seb-jean** [comment]
I made this addition because the icon has the `xmlns` attribute defined at the start but it is not rendered at the end.

**Review by smnandre — APPROVED** [review]

**Comment by norkunas** [comment]
> I made this addition because the icon has the `xmlns` attribute defined at the start but it is not rendered at the end.

But is it needed? :slightly_smiling_face:

**Comment by seb-jean** [comment]
> > I made this addition because the icon has the `xmlns` attribute defined at the start but it is not rendered at the end.
> 
> But is it needed? 🙂

Do you think this is an attribute that is optional?

**Comment by norkunas** [comment]
Yes, browsers renders svg's without it perfectly fine :)

**Comment by seb-jean** [comment]
Well then, I don't know if it's an optional or required attribute 😄.

**Comment by smnandre** [comment]
It is "required" but in reality for this kind usage it makes no difference.

**Review by javiereguiluz — APPROVED** [review]

**Comment by kbond** [comment]
I think this clearly states it isn't required for ux-icons:

> Note: The xmlns attribute is only required on the outermost svg element of SVG documents, or inside HTML documents with XML serialization. It is unnecessary for inner svg elements or inside HTML documents with HTML serialization.
> -- https://developer.mozilla.org/en-US/docs/Web/SVG/Element/svg

**Comment by seb-jean** [comment]
I am closing this PR for the reasons given by @kbond.

**Pull request closed without merging**